client/components/data/query-plans/index.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingPlans } from 'state/plans/selectors';
+import { requestPlans } from 'state/plans/actions';
+
+class QueryPlans extends Component {
+	componentWillMount() {
+		if ( ! this.props.requestingPlans ) {
+			this.props.requestPlans();
+		}
+	}
+
+	render() {
+		return null;
+	}
+}
+
+QueryPlans.propTypes = {
+	requestingPlans: PropTypes.bool,
+	requestPlans: PropTypes.func
+};
+
+QueryPlans.defaultProps = {
+	requestPlans: () => {}
+};
+
+export default connect(
+	state => {
+		return {
+			requestingPlans: isRequestingPlans( state )
+		};
+	},
+	dispatch => {
+		return bindActionCreators( { requestPlans }, dispatch );
+	}
```

Comment:
We can pass in an object as the second argument. See [docs](https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options) This block can be simplified down to:
```javascript
state => {
 		return {
 			requestingPlans: isRequestingPlans( state )
 		};
 	},
 { requestPlans }
```